src/java/org/apache/cassandra/tcm/serialization/Version.java

```diff
@@ -37,6 +37,7 @@ public enum Version
      *  - Added version to PlacementForRange serializer
      *  - Serialize MemtableParams when serializing TableParams
      *  - Added AccordFastPath
+     *  - Added StaleReplicas
```

Comment:
Just a note: if there are any deployments out there, adding elements without actually bumping the version is likely to break them. Of course, this is not a released version yet so there isn't any guarantee that we won't do that but we will need to bump it prior to merging to trunk.

Reply:
So `AccordFastPath` and `StaleReplicas` are both new in `cep-15-accord` should we, before merging to trunk, assign them both to a new V3?

Reply:
CC @bdeggleston